downstream/modules/aap-hardening/proc-configure-centralized-logging.adoc

```diff
@@ -80,7 +80,7 @@ To set up logging to any of the aggregator types, read the documentation on the
 . When done, click *Save* to apply the settings or Cancel to abandon the changes.
```

Comment:
I think these are buttons and should be tagged as "Click btn:[Save]" or "btn:[Cancel]". At a minimum the UI selections in this section should be bolded.